src/Sentry.Unity/Integrations/SessionIntegration.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using Sentry.Extensibility;
+using Sentry.Infrastructure;
+using Sentry.Integrations;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Sentry.Unity.Integrations
+{
+    public class SessionIntegration : ISdkIntegration
+    {
+        public SessionIntegration()
+        { }
+
+        public void Register(IHub hub, SentryOptions options)
+        {
+            if (!options.AutoSessionTracking)
+            {
+                return;
+            }
+
+            options.DiagnosticLogger?.LogDebug("Registering Session integration.");
+
+            var gameListenerObject = new GameObject("SentryListener");
```

Comment:
Do we need to check if the object already created?

Example: user configured via Sentry window approach, but then decided to use a programmatic one without disabling window - 2 game objects in the scene. Or we don't support such cases?